WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt

```diff
@@ -999,6 +999,19 @@ d_milan = {
 	}
 }
 
+d_perusia = 
+    color={ 255 291 130 }
+	color2={ 255 255 255 }
+	
+	capital = 322
+	
+	culture = lombard
+	
+	allow = {
+		always = no
+	}
+}
+
```

Comment:
This title does not have any flag.

Reply:
How do I exactly set that?

Reply:
You add one in the WTWSMS\gfx\flags folder. See the wiki for more information: https://ck2.paradoxwikis.com/Title_modding#Adding_a_coat_of_arms

Reply:
Ohh, I thought it was like an event flag not a gfx flag